.gitignore

```diff
@@ -0,0 +1 @@
+/.idea
```

Comment:
Shouldn't this be in an individuals global gitignore?

Reply:
Right, deleted gitignore for now